app/src/main/java/com/codesoom/assignment/dto/SessionResponseData.java

```diff
@@ -0,0 +1,15 @@
+package com.codesoom.assignment.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * 로그인 후 생성된 JWT DTO
+ */
```

Comment:
> Typically the data that crosses the boundaries is simple data structures. You can use basic structs or simple Data Transfer objects if you like. Or the data can simply be arguments in function calls. Or you can pack it into a hashmap, or construct it into an object. The important thing is that isolated, simple, data structures are passed across the boundaries. We don’t want to cheat and pass Entities or Database rows. We don’t want the data structures to have any kind of dependency that violates The Dependency Rule.
>
> For example, many database frameworks return a convenient data format in response to a query. We might call this a RowStructure. We don’t want to pass that row structure inwards across a boundary. That would violate The Dependency Rule because it would force an inner circle to know something about an outer circle.
>
>So when we pass data across a boundary, it is always in the form that is most convenient for the inner circle.

DTO가 무엇인지 자세히 공부하고 넘어가도 좋아요

See also
  - https://blog.cleancoder.com/uncle-bob/2012/08/13/the-clean-architecture.html